src/components/RefinementList/RefinementList.js

```diff
@@ -146,65 +143,103 @@ export class RawRefinementList extends Component {
 
   render() {
     // Adding `-lvl0` classes
-    const cssClassList = [this.props.cssClasses.list];
-    if (this.props.cssClasses.depth) {
-      cssClassList.push(`${this.props.cssClasses.depth}${this.props.depth}`);
-    }
+    const cssClassList = cx(this.props.cssClasses.list, {
+      [`${this.props.cssClasses.depth}${this.props.depth}`]: this.props
+        .cssClasses.depth,
+    });
+
+    const showMoreButtonClassName =
+      this.props.showMore === true && this.props.canToggleShowMore
+        ? this.props.cssClasses.showMore
+        : cx(
+            this.props.cssClasses.showMore,
+            this.props.cssClasses.disabledShowMore
+          );
 
-    const showMoreBtn =
-      this.props.showMore === true && this.props.canToggleShowMore ? (
+    const showMoreButton =
+      this.props.showMore === true ? (
         <Template
-          rootProps={{ onClick: this.props.toggleShowMore }}
+          rootTagName="button"
           templateKey={`show-more-${
             this.props.isShowingMore ? 'active' : 'inactive'
           }`}
+          rootProps={{
+            className: showMoreButtonClassName,
+            onClick: this.props.toggleShowMore,
+          }}
           {...this.props.templateProps}
         />
-      ) : (
-        undefined
-      );
+      ) : null;
 
-    const shouldDisableSearchInput =
+    const shouldDisableSearchBox =
       this.props.searchIsAlwaysActive !== true &&
       !(this.props.isFromSearch || !this.props.hasExhaustiveItems);
-    const searchInput = this.props.searchFacetValues ? (
-      <SearchBox
-        ref={i => {
-          this.searchbox = i;
-        }}
-        placeholder={this.props.searchPlaceholder}
-        onChange={this.props.searchFacetValues}
-        onValidate={() => this.refineFirstValue()}
-        disabled={shouldDisableSearchInput}
-      />
+    const searchBox = this.props.searchFacetValues ? (
+      <div className={this.props.cssClasses.searchBox}>
+        <SearchBox
+          ref={i => {
+            this.searchbox = i;
+          }}
+          placeholder={this.props.searchPlaceholder}
+          onChange={this.props.searchFacetValues}
+          onValidate={() => this.refineFirstValue()}
+          disabled={shouldDisableSearchBox}
+        />
+      </div>
     ) : null;
 
+    const facetValues =
+      this.props.facetValues && this.props.facetValues.length > 0 ? (
+        <ul className={cssClassList}>
+          {this.props.facetValues.map(this._generateFacetItem, this)}
+        </ul>
+      ) : null;
+
     const noResults =
       this.props.searchFacetValues &&
       this.props.isFromSearch &&
       this.props.facetValues.length === 0 ? (
```

Comment:
Since the second part of the ternary is null we can use the `&&` logical operator.